src/reducers/real/index.js

```diff
@@ -0,0 +1,90 @@
+import {
+  ITEM_ADD,
+  ITEM_DELETE,
+  ITEM_RENAME,
+  CONTAINER_ADD,
+  CONTAINER_DELETE,
+  CONTAINER_RENAME,
+  SET_REAL_DATA
+} from 'actions/actiontypes'
+
+const initialState = {
+  _id: '',
+  name: '',
+  items: [],
+  modified_timestamp: '',
+  containers: [],
+  is_deleted: false,
+  timestamp: '',
+  snapshots: [],
+  unsnapshot_items: []
+}
+
+const realReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ITEM_ADD:
+      return {
+        ...state,
+        items: [
+          ...state.items,
+          action.item
+        ],
+        unsnapshot_items: [
+          ...state.unsnapshot_items,
+          action.item
+        ]
+      }
+    case ITEM_DELETE:
+      let items = state.items
+      items = items.filter(ele => ele._id !== action.id)
```

Comment:
so the problem with maintaining an "_unassigned" list means that when we delete the item, it needs to be removed everywhere.

Reply:
So let's make sure to test this as well.

Reply:
just want to make sure that I understand what we are testing, you want me to test that ITEM_delete actually removes the item id.

Reply:
Yes i want to make sure that when u delete an item, it deletes it across all the snapshots.